frontend/__tests__/register/register.spec.tsx

```diff
@@ -1,73 +0,0 @@
-import { render, screen, waitFor } from '@testing-library/react';
```

Comment:
i deleted this file as well as the other register files in the frontend since the register flow will be completely different from what was in the template